main/spl/APKBUILD

```diff
@@ -19,6 +19,7 @@ builddir="$srcdir"/spl-$pkgver
 
 build() {
 	cd "$builddir"
+	autoreconf -fi
```

Comment:
Please move `autoreconf -fi` to `prepare()` function:

```sh
prepare() {
 	default_prepare || return 1
 	autoreconf -fi
}
```

and add comment why it’s needed to run `autoreconf` despite the `configure` script exists.